base/testfiles/tlb-picture-01.lvt

```diff
@@ -0,0 +1,42 @@
+\documentclass{article}
+\input{test2e}
+
+
+% test new length syntax does not error and
+% generally improve coverage of picture mode in test suite
+
+
+\begin{document}
+
+\newlength\zzz
+\newlength\zzzb
+\setlength\zzz{100pt}
+
+\START
+
+\begin{picture}(200,4\zzz)
+
+\loop
+\addtolength\zzzb{40pt}%
+\put(\zzzb+30pt,\zzzb+40pt){\line(1,0){150pt}}
+\put(\zzzb+30pt,\zzzb+40pt){\line(0,1){60}}
+\put(\zzzb+30pt,\zzzb+\zzz){\line(1,0){20}}
+\put(\zzzb+50pt,\zzzb+\zzz){\line(1,-4){10}}
```

Comment:
there should also be a test file testing the version with plain numbers and posibly with mixtures

Reply:
> 
> 
> there should also be a test file testing the version with plain numbers and posibly with mixtures

Yes although there were already half a dozen picture environments in tests, plus more in the documentation using the existing syntax so those tests not breaking already covers the existing syntax to some extent.